internal/distro/rhel9/images.go

```diff
@@ -340,6 +340,10 @@ func edgeRawImage(workload workload.Workload,
 		Checksum:   options.OSTree.FetchChecksum,
 	}
 	img := image.NewOSTreeRawImage(commit)
+	// TODO: add Fedora once it's ready
+	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || t.arch.distro.osVersion == "9-stream" {
+		img.Ignition = true
+	}
```

Comment:
Can we separate this part (and the one below for the installer) which is a bugfix from the rest of the commit that only mentions adding a test?

Reply:
sure I can!

Reply:
fixed!